api/src/main/java/com/capitalone/dashboard/auth/AuthProperties.java

```diff
@@ -0,0 +1,70 @@
+package com.capitalone.dashboard.auth;
+
+import java.util.UUID;
+
+import javax.annotation.PostConstruct;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "auth")
+public class AuthProperties {
+	
+	private static final Logger LOGGER = LoggerFactory.getLogger(AuthProperties.class);
+	
+	private Long expirationTime;
+	private String secret;
```

Comment:
If you don't provide one, a random one would be generated, this works fine unless you have multiple instances of the API running.